Examples/ObjectiveC/ExtrusionsExample.m

```diff
@@ -0,0 +1,56 @@
+//
+//  ExtrusionsExample.m
+//  Examples
+//
+//  Created by Jordan Kiley on 5/11/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+#import "ExtrusionsExample.h"
+
+@import Mapbox;
+
+NSString *const MBXExample3DExtrusions = @"ExtrusionsExample";
+
+@interface ExtrusionsExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ExtrusionsExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
+    
+    // Center the map on the Colosseum in Rome, Italy.
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(41.8902, 12.4922)];
```

Comment:
Actually, we’re setting the camera right below. Let’s combine the two statements.